index.html

```diff
@@ -14,22 +14,34 @@
 </head>
 
 <body class="portfolio">
-  <div class="main">
+  <main>
     <nav class="app-bar">
       <a class="My-logo" tabindex="1">My Logo</a>
-      <a class="hamburguer"><img src="img/Enabled.png" alt="sub-menu" class="H-iamge"></a>
+      <button class="hamburguer"></button>
       <ul class="dsk-menu">
         <li class="l-dskmenu">
-          <p tabindex="2">Portfolio</p>
+          <p class="m-menu" tabindex="2">Portfolio</p>
         </li>
         <li class="l-dskmenu">
-          <p tabindex="3">About</p>
+          <p class="m-menu" tabindex="3">About</p>
         </li>
-        <li class="l-dskmenu">
+        <li class="m-menu" class="l-dskmenu">
           <p tabindex="4">Contact</p>
         </li>
       </ul>
     </nav>
+    <ul class="mbl-menu">
+      <li id="close"><button id="btn-close" alt="close menu"><img src="img/x.png" class="img-close"></button></li>
+      <li class="l-dskmenu">
+        <p class="m-menu" tabindex="2">Portfolio</p>
+      </li>
+      <li class="l-dskmenu">
+        <p class="m-menu" tabindex="3">About</p>
+      </li>
+      <li class="m-menu" class="l-dskmenu">
+        <p tabindex="4">Contact</p>
+      </li>
+    </ul>
```

Comment:
- [ ] Good job so far @Team, you have done a good job so far. However, according to the [Project requirements,](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) you should also implemented these:
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears. 👍 
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. 👍 

- I have attached a video below for your reference 💯 

https://user-images.githubusercontent.com/102699958/232846083-234a99d1-1132-4c61-a7ba-4b50e48b4a52.mov